salvo/src/lib/common/file_ops.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
Sorry I must have missed this - can we also add tests for the `file_ops` package?

Reply:
Added a test for file_ops